src/TagLib/Ogg/XiphComment.cs

```diff
@@ -48,17 +48,23 @@ public class XiphComment : TagLib.Tag, IEnumerable<string>
 			new Dictionary<string, string[]> ();
 
 		/// <summary>
-		///    Contains the ventor ID.
+		///    Contains the vendor ID.
 		/// </summary>
 		private string vendor_id;
 
+		/// <summary>
+		///    Saves BeatsPerMinute tag as either "Tempo" or "BPM"
+		///    based on which was last read.
+		/// </summary>
+		private static bool save_BPM_as_tempo = true;
+
 		/// <summary>
 		///    Picture instances parsed from the fields.
 		/// </summary>
 		private IPicture[] pictures = null;
 
 		/// <summary>
-		///    true if the picture fields ni <see cref="field_list" />
+		///    true if the picture fields in <see cref="field_list" />
 		///    should be updated from the <see cref="pictures"/> array.
```

Comment:
These typos are great fixes. Where it makes sense, it is nice to have them in a separate PR. They are super simple to review and merge :)